source/hardware/controllers/hercules_djcontrol_mp3_e2.rst

```diff
@@ -3,589 +3,123 @@ Hercules DJControl MP3 e2 / MP3 LE / Glow
-Headphone volume                                                  Shift + Deck A pitch knob 3
-Adjust the cue/main mix in the headphone output                   Shift + Deck B pitch knob 3
-Toggles deck output to the headphones monitor on/off              Headphone monitor         20
-================================================================= ========================= ======
+Mapping description (by function)
+---------------------------------
 
-Sampler
+Library
 ^^^^^^^
 
-.. figure:: ../../_static/controllers/hercules_djcontrol_mp3_e2_sampler.jpg
-   :align: center
-   :width: 100%
-   :figwidth: 100%
-   :alt: Hercules DJControl MP3 e2 (sampler)
-   :figclass: pretty-figures
-
-   Hercules DJControl MP3 e2 (sampler)
-
-
-
-=========================================================================== =================== ======
-Function                                                                    Control             number
-=========================================================================== =================== ======
-Loads the current highlighted track into the corresponding sampler (1 or 2) Shift + Load A/B    18
-Goto start & Play sampler 1/3                                               Shift + Folder      6
-Stop sampler 1/3                                                            Supershift + Folder 6
-Goto start & Play sampler 2/4                                               Shift + Files       10
-Stop sampler 2/4                                                            Supershift + Files  10
-=========================================================================== =================== ======
++----------------------------------------------------------------------------+------------+--------+
+| Function                                                                   | Control    | Number |
++============================================================================+============+========+
+| Toggle playlist selection                                                  | Folder     | 6      |
++----------------------------------------------------------------------------+------------+--------+
+| Toggle track selection                                                     | Files      | 10     |
++----------------------------------------------------------------------------+------------+--------+
+| Go one playlst/track up                                                    | Up arrow   | 8      |
++----------------------------------------------------------------------------+------------+--------+
+| Go one playlst/track down                                                  | Down arrow | 8      |
++----------------------------------------------------------------------------+------------+--------+
+| Loads the currently highlighted track into the corresponding deck (A or B) | Load A/B   | 18     |
++----------------------------------------------------------------------------+------------+--------+
```

Comment:
Please use the simpler table formatting (as in `=`) and ideally use the following column order: Number, Control, Function

Also, if a control is *labeled* on the controller itself, please use the hwlabel directive, e.g.:

    :hwlabel:`CUE`